src/main/java/de/uhd/ifi/se/decision/management/jira/webhook/WebhookContentProviderForTreant.java

```diff
@@ -16,37 +16,56 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeElement;
 import de.uhd.ifi.se.decision.management.jira.view.treant.Treant;
 
 /**
  * Creates the content submitted via the webhook. The content consists of a key
  * value pair. The key is an issue id. The value is the Treant JSON String.
  */
-public class WebhookContentProvider {
+public class WebhookContentProviderForTreant {
 
 	private String projectKey;
 	private String rootElementKey;
 	private String secret;
+	private KnowledgeElement knowledgeElement;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'knowledgeElement' could be replaced by a local variable.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5269053)